ClientApp/src/components/CardPage.jsx

```diff
@@ -44,6 +57,8 @@ export class CardPage extends Component {
         this.toggleRobinHood = this.toggleRobinHood.bind(this);
         this.toggleStealGem = this.toggleStealGem.bind(this);
         this.toggleStealCard = this.toggleStealCard.bind(this);
+        this.toggleDoubleReceive = this.toggleDoubleReceive.bind(this);
```

Comment:
You already binded this in the callback as well as Malediction. Only bind once, either in the callback or up here.